sphinx/ext/imgmath.py

```diff
@@ -197,8 +197,17 @@ def convert_dvi_to_svg(dvipath, builder):
     command.extend(builder.config.imgmath_dvisvgm_args)
     command.append(dvipath)
 
-    convert_dvi_to_image(command, name)
-    return filename, None
+    stdout, stderr = convert_dvi_to_image(command, name)
+
+    depth = None
+    if builder.config.imgmath_use_preview:
+        for line in stderr.splitlines():  # not stdout !
```

Comment:
The dvisvgm output (as shown above) goes to stderr... whereas the dvipng used stdout...